### PR TITLE
Update location of Spring Authorization Server

### PR DESCRIPTION
**Pull request by sjohnr**

### CODE REVIEW

**Comment by wilkinsona** [comment]
This is changing Spring Native which, judging by the PR's title, isn't what you intended.

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -279,7 +279,7 @@ issuebot:
           - jzheaux
           - rwinch
           - sjohnr
-      - organization: spring-projects-experimental
```
Comment (wilkinsona) [inline src/main/resources/application.yml]: Should this have been a change to line 274 instead?

**Review by wilkinsona — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yml (hunk 1 above)
Comment (sjohnr) [reply src/main/resources/application.yml]: Oops. Thanks for catching that! I'll update it.

**Comment by sjohnr** [comment]
Thanks, I've updated the PR.

**Pull request closed without merging**